src/MCSSolver.py

```diff
@@ -39,31 +41,45 @@ def __init__(self):
       @ Out, None
     """
     ExternalModelPluginBase.__init__(self)
+    self.TimeDependentMode= True
```

Comment:
self.TimeDependentMode --> self.timeDependentMode
Also, please provide inline comments for this variable.